Makefile

```diff
@@ -81,7 +81,7 @@ dc-build:
 	COMPOSE_DOCKER_CLI_BUILD=1 DOCKER_BUILDKIT=1 docker-compose build
 
 
-# remove docker containers, volumes, images left by existing system, remove vendor folders, rebuild everything
+# remove docker containers, volumes, images left by existing system, rebuild everything
```

Comment:
How come this is now not removing the vendor folders?

Reply:
It's pointless step, it just adds time on to any rebuilds locally. Any changes to the base files would be detected when composer is run during the build/up steps.
 
I've been meaning to remove this for ages & it was driving me nuts when I was doing rebuilds, adding another 5-10min (Albeit I should have targeted containers I was doing rather than full rebuild)